### ISSUE TITLE
When building incorrect files are sought resulting in failed build.

### ISSUE THREAD

**Issue by sec6** [zynaddsubfx/zyn-fusion-build#21]
Script calls for **libuv.a** when it should be **libuv-win.a**.  Script calls for **libuv-win.a** when it should be **libuv.a**.  As a test, I renamed files during build and the build was successful.  It is going to the right directory but is requesting the incorrect file.  Libuv.a and libuv-win.a are mixed up.

**Comment by fluzz142857** [comment]
Yes, I had to do the same thing. I pasted the files in while the script was running and it built just fine.

**Comment by sec6** [comment]
I tried it the way you did at first to get it to build, but there were problems with the vst.  It is going to the right directory but calling the wrong files.  When it wants libuv.a, it really wants libuv-win.a and when it calls for libuv-win.a, it really needs libuv.a. What you need to do is when the script is running is rename the files.

**Comment by fluzz142857** [comment]
What types of problems were you having with the VST? I was having issues too and it might be because I should have renamed them.

**Comment by sec6** [comment]
If I remember correctly, the GUI of the vst wouldn't load properly.

**Comment by fluzz142857** [comment]
Wow, that's exactly the issue I was having. I'll try renaming them instead

**Comment by fluzz142857** [comment]
I renamed the files in the build script by adding
```
mv deps/libuv-v1.9.1/.libs/libuv.a deps/libuv-v1.9.1/.libs/libuv-win.a
mv deps/libuv-win.a deps/libuv.a
```
[https://github.com/fluzz142857/zyn-fusion-build/blob/master/build-mingw64.sh]((https://github.com/fluzz142857/zyn-fusion-build/blob/master/build-mingw64.sh)). Build worked but GUI still failed.

**Comment by sec6** [comment]
Not sure why it is not working. As I said, what I did was just rename the files and it worked fine.  I see you hit the error with python as well.  I installed the version of python it was looking for before doing the build.  Maybe this makes a difference. I am not a programmer and know very little about this stuff.

**Comment by fluzz142857** [comment]
Ok, thanks. If you don't mind and still have the files, could you upload your zyn-fusion-build-master or mruby-zest-build folders? 

Also, just to clarify, you switched the names of libuv.a and libuv-win.a, correct?

Thank you

**Comment by fluzz142857** [comment]
All right, thanks for your help.

**Comment by fluzz142857** [comment]
To solve the libuv.a and libuv-win.a, I added:
```
mv deps/libuv-v1.9.1/.libs/libuv.a deps/libuv-v1.9.1/.libs/libuv-win.a
mv deps/libuv-win.a deps/libuv.a
```
to lines 89 and 90 of build-mingw64.sh. Does your solution remove the need for this?

**Comment by antanova** [comment]
Yes, although I was using `build-windows.rb` rather than `build-mingw64.sh`. I used an Ubuntu VM to do my building. What my solution aims to do is to get those lines to refer to only one file - whereas I think if you don't change the Makefile, you end up referring to 2 different but identical files. I did that for one build and the GUI wouldn't respond, which is why I ended up with the solution I tried.
